includes/cli/class-setup.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/**
+ * Newspack setup wizard. Required plugins, introduction, and data collection.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack\CLI;
+
+use WP_CLI;
+use WP_REST_Request;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Setup Newspack CLI command.
+ */
+class Setup {
+
+	/**
+	 * Sets up a testing site
+	 *
+	 * This is a command used to set up a testing site from scratch, installing required plugins and running the onboard wizard automatically
+	 *
+	 * ## OPTIONS
+	 * [--skip-ras]
+	 * : If set, will not add the EXPERIMENTAL RAS flag to wp-config
+	 *
+	 * @param array $args Positional arguments.
+	 * @param array $assoc_args Assoc arguments.
+	 * @return void
+	 */
+	public function __invoke( $args, $assoc_args ) {
+		do_action( 'rest_api_init' );
+		wp_set_current_user( 1 );
```

Comment:
This assumes user of ID `1` is the admin. The command should check if that's true first, and perhaps create an admin user with an environment-defined username.

Reply:
I updated it to find the first admin.

But as I said, I'm not too worried about this as the scope is only for testing sites. For example, we also assume wp-config.php is placed in the ABSPATH, which is not true for every site.